imops/interp2d.py

```diff
@@ -81,7 +81,7 @@ def __call__(self, points: np.ndarray, values: np.ndarray = None, fill_value: fl
         points: np.ndarray
             2-D array of data point coordinates to interpolate at
         values: np.ndarray
-            1-D array of fp32/fp64 values to use at initial points. If passed, existing values will be rewritten
+            1-D array of fp32/fp64 values to use at initial points. If passed, existing values will NOT be rewritten
```

Comment:
"If passed, existing values will NOT be rewritten" - you don't need to specify this. This is what the user expects anyway

Reply:
fixed